command/ssh/login.go

```diff
@@ -187,6 +200,18 @@ func loginAction(ctx *cli.Context) error {
 		return err
 	}
 
+	// NOTE: For OIDC tokens the subject should always be the email. The
+	// provisioner is responsible for loading and setting the principals with
+	// the application of an Identity function.
+	if email, ok := tokenHasEmail(token); ok {
```

Comment:
it's strange to have `tokenHasEmail` and `tokenSubject` when they return the same thing (`(val, bool)`). Let's use the same naming convention. I like `tokenSubject` and `tokenEmail`.

Reply:
Fixed with 2c2a8b5